targets/riscv64.json

```diff
@@ -0,0 +1,14 @@
+{
+	"inherits": ["riscv"],
+	"llvm-target": "riscv64--none",
+	"build-tags": ["tinygo.riscv64"],
+	"code-model": "medium",
```

Comment:
Should this be left as default, or do all 64-bit RISC-V systems need this?

Reply:
No you are right, this belongs in the chip's definition since it may not always be needed.